superset/translations/pt_BR/LC_MESSAGES/messages.po

```diff
@@ -3961,36 +3958,35 @@ msgstr ""
 
 #: superset/views/core.py:480
 msgid "Owners is a list of users who can alter the dashboard."
-msgstr ""
-"Os proprietários são uma lista de usuários que podem alterar o painel."
+msgstr "Os proprietários são uma lista de usuários que podem alterar o painel."
 
 #: superset/views/core.py:488 superset/views/core.py:555
 msgid "Dashboard"
-msgstr "painel de controle"
+msgstr "Painel de Controle"
 
 #: superset/views/core.py:490
 msgid "Slug"
-msgstr "Slug"
+msgstr "Lesma"
```

Comment:
Does it make sense?

Reply:
Is it a "feature" or something like this? If yes we can keep in English. But yes, this are correct translation.